src/main/java/bio/terra/service/snapshotbuilder/utils/ConceptChildrenQueryBuilder.java

```diff
@@ -60,7 +62,7 @@ public Query buildConceptChildrenQuery(
     // domain specific occurrence table joined on concept_ancestor.descendant_concept_id =
     // 'domain'_concept_id
     TableVariable domainOccurrence =
-        TableVariable.forJoined(
+        TableVariable.forLeftJoined(
```

Comment:
which ticket is this change for?

Reply:
Is this to include things that have a 0 count?

Reply:
I did this while testing to see the proper results, although it is in @evansuslovich 's  PR too. I could revert this case if it makes it easier to merge the two PRs..  although there will be a lot of overlap and conflicts to resolve either way.